### DISCUSSION TITLE
Video Reader

### DISCUSSION THREAD

**Discussion by LRCodex** [QL-Win/QuickLook#1364] [Q&A]
Hello,

I am dealing with 120fps 4k videos in h264 (high 4:2:2) and believe me when I say this: not a single video reader can handle them. VLC buffers so much that you have to play the video in slow motion to see it (even after optimising the vlc settings for these videos). The same goes for all the other software I have explored. They all fail miserably at reading the video smoothly. Tried it on different high end laptops with the same problem. 
Until one day I discovered that Quicklook could play them smoothly, as if they were nothing. This is truly amazing, and it would be really helpful for me to understand how it could succeed where all the specialised software has failed? How does it handle video codecs so well, and could I learn something from it to apply to my parameters in VLC?

Thank you so much.

**Comment by xupefei** [comment]
My guess is VLC does not HW decoding enabled.
QuickLook is using [LAVfilters](https://github.com/Nevcairiel/LAVFilters/releases) with HW decoding enabled. Give it a try (LAVfilters is shipped with MPC player)?
